src/SFA.DAS.Reservations.Infrastructure/Api/ApiClient.cs

```diff
@@ -16,14 +17,30 @@ public ApiClient(IOptions<ReservationsApiConfiguration> apiOptions)
             _apiOptions = apiOptions;
         }
 
-        public async Task<string> GetReservations()
+        public async Task<string> GetReservations(long accountId)
         {
             var accessToken = await GetAccessTokenAsync();
             using (var client = new HttpClient())//not unit testable using directly
             {
                 client.DefaultRequestHeaders.Authorization = new AuthenticationHeaderValue("Bearer", accessToken);
 
-                var response = await client.GetAsync($"{_apiOptions.Value.Url}api/accounts/1/reservations").ConfigureAwait(false);
+                var response = await client.GetAsync($"{_apiOptions.Value.Url}api/accounts/{accountId}/reservations").ConfigureAwait(false);
+
+                response.EnsureSuccessStatusCode();
+
+                return await response.Content.ReadAsStringAsync().ConfigureAwait(false);
+            }
+        }
+
+        public async Task<string> CreateReservation(long accountId, string json)
+        {
+            var accessToken = await GetAccessTokenAsync();
+            using (var client = new HttpClient())//not unit testable using directly
```

Comment:
Can we whack an interface around this? Or pull in the SFA.DAS.SecureHttp package?